build/version.props

```diff
@@ -1,7 +1,7 @@
 <!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>1.1.0</VersionPrefix>
+    <VersionPrefix>1.2.0</VersionPrefix>
```

Comment:
Not sure how to think about versioning on this type of templates package, since you never really "update" anything. But perhaps a new major .NET version warrants a new major of the templates?

Reply:
Personally I prefer having the same major version of a project template as the major version of the framework it’s intended for. So let’s just bump it 11 versions 😜

Reply:
We do however also have Commerce, which is not the same major as CMS. I think I will keep 1.2.0 for now.